docs/add_authenticable_action.md

```diff
@@ -0,0 +1,21 @@
+# How to add a new authenticable action?
+## Proposals example
+We're going to reproduce the steps to add an action (adhere) for a proposal step by step.
+### Configuring a new 'adhere' action
+1. Edit decidim-proposals/lib/decidim/proposals/feature.rb
+1. Add the new 'adhere' action into the `feature.actions` array and save the file:
+```
+feature.actions = %w(adhere vote create)
+```
+
+3. Translate the action for the corresponding key: `en.decidim.features.proposals.actions.adhere= Adhere`
+1. Restart the server to pick up the changes.
+1. Now the admin should be able to go to the Control Panel and edit `PROCESSES/Proposals/Permissions/Adhere` panel. There an `Authorization Handler` can be set.
+1. Edit `app/models/decidim/proposals/abilities/current_user_ability.rb` and add the corresponding cancancan ability like `can :adhere, Proposal do |_proposal| ...`.
```

Comment:
You'll need to restart the server after changing this, I think :confused: Just to be sure, can you move this step before the one to restart the server?